kotlin-redux/src/main/kotlin/redux/KotlinEnhancer.kt

```diff
@@ -4,9 +4,7 @@ import kotlinext.js.js
 
 fun <S> kotlinEnhancer(next: StoreCreator<S, WrapperAction>): StoreCreator<S, KotlinAction> {
     return { reducer, preloadedState ->
-        @Suppress("NAME_SHADOWING") // Needed to allow replaceReducer to mutate reducer
-        var reducer = reducer
-        val wrapperReducer: Reducer<S, WrapperAction> = { state, action -> reducer(state, action.action) }
+        var wrapperReducer: Reducer<S, WrapperAction> = { state, action -> reducer(state, action.action) }
```

Comment:
This is a nit, but I'd prefer extracting a top-level function to introducing a mutable variable:

```
val wrapReducer: (Reducer<S, KotlinAction>) -> Reducer<S, WrapperAction> { reducer ->
    { state, action -> reducer(state, action.action) }
}
```